### ISSUE TITLE
Request for Charts in SVG Format

### ISSUE THREAD

**Issue by jiangshengdev** [Tongji-KGLLM/RAG-Survey#13]
I'm very interested in your RAG-Survey project and find that having the charts in SVG format would greatly aid my learning and usage. I'm looking to translate some content into Chinese, and vector graphics would be much more convenient for this purpose.

Could you please share the SVG files of these charts? Such resources would be incredibly valuable to me.

Thank you

**Comment by yunfan42** [comment]
Of course. Please leave me an email address, and I will email the SVG format image to you.

**Comment by jiangshengdev** [comment]
My email is [jiangshengdev@outlook.com](mailto:jiangshengdev@outlook.com). Please send the SVG image there. Thanks!

**Comment by yunfan42** [comment]
Dear Mr Sheng. 
Please check the SVG Image in email attachments.
Please credit the source when using it. 
Thank you.



| |
Yunfan Gao
|


---- Replied Message ----
| From | Jiang ***@***.***> |
| Date | 04/7/2024 09:11 |
| To | ***@***.***> |
| Cc | ***@***.***> ,
***@***.***> |
| Subject | Re: [Tongji-KGLLM/RAG-Survey] Request for Charts in SVG Format (Issue #13) |

My email is ***@***.*** Please send the SVG image there. Thanks!

**Comment by jiangshengdev** [comment]
First and foremost, I would like to express my sincere gratitude for your prompt response to my request.

Unfortunately, upon checking my inbox at [jiangshengdev@outlook.com](mailto:jiangshengdev@outlook.com), I have not been able to locate the SVG chart attachments that were mentioned. It seems there may have been a technical hiccup or perhaps an issue with the email filtering system.

To circumvent any further inconvenience, I kindly ask if you could resend the SVG attachments to my new email address: [284218048@qq.com](mailto:284218048@qq.com). I apologize for any additional effort this may require on your part and deeply appreciate your continued assistance.

I assure you that all materials received will be used with the utmost respect for copyright and proper attribution, as per your guidance.

Thank you once again for your time and understanding. I am looking forward to your reply.
